File: BMI.js
// bmi calculator
// input current weight 
let w = parseFloat(prompt("Enter your weight in pounds: "));
let h = parseFloat(prompt("Enter your height in inches: "));
let bmi = w / (h * h) * 703;
console.log("Your BMI is " + bmi.toFixed(2));

File: BMI.java
import java.util.Scanner;
public class BMI {

    // class-level variables
    private static double w;
    private static double h;
    private static double bmi;

    public static void main(String[] args) {
        calc();
    }
    public static void calc(){
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter your weight in pounds: ");
        // take the users weight in pounds
        w = scanner.nextDouble();
        System.out.println("Enter your height in inches: ");
        // take the users height in inches
        h = scanner.nextDouble();
        // calculate the users BMI
        bmi = (w * 703) / (h * h);
        System.out.println("Your BMI is: " + bmi);
    }
}
